Web UI frontend fails to type-check with current dependencies

## Problem

The Web UI frontend fails type-checking with the currently resolved dependency versions.

## Reproduction

From examples/web_ui/frontend on current main, run:

```bash
./node_modules/.bin/tsc -b
```

The first blocking error is:

```text
tsconfig.app.json(25,3): error TS5101: Option 'baseUrl' is deprecated and will stop functioning in TypeScript 7.0. Specify compilerOption '"ignoreDeprecations": "6.0"' to silence this error.
```

After temporarily passing `--ignoreDeprecations 6.0`, type-checking continues and reports:

```text
src/components/ui/calendar.tsx(77,5): error TS2353: Object literal may only specify known properties, and 'table' does not exist in type 'Partial<ClassNames>'.
src/hooks/useSkills.ts(3,10): error TS2305: Module '"../api"' has no exported member 'skillApi'.
```

## Proposed fix

- Use the react-day-picker v10 `month_grid` classNames key.
- Use the existing `workspaceApi.skill` client for skills operations.
- Configure TypeScript 6 deprecation handling in the frontend tsconfig.

## Validation

- `./node_modules/.bin/tsc -b`
- `./node_modules/.bin/vite build`

--- a/examples/web_ui/frontend/src/components/ui/calendar.tsx
+++ b/examples/web_ui/frontend/src/components/ui/calendar.tsx
@@ -74,7 +74,7 @@ function Calendar({
 						: 'flex items-center gap-1 rounded-(--cell-radius) text-sm [&>svg]:size-3.5 [&>svg]:text-muted-foreground',
 					defaultClassNames.caption_label,
 				),
-				table: 'w-full border-collapse',
+				month_grid: cn('w-full border-collapse', defaultClassNames.month_grid),
 				weekdays: cn('flex', defaultClassNames.weekdays),
 				weekday: cn(
 					'flex-1 rounded-(--cell-radius) text-[0.8rem] font-normal text-muted-foreground select-none',

--- a/examples/web_ui/frontend/src/hooks/useSkills.ts
+++ b/examples/web_ui/frontend/src/hooks/useSkills.ts
@@ -1,6 +1,6 @@
 import { useState, useEffect, useCallback } from 'react';
 
-import { skillApi } from '../api';
+import { workspaceApi } from '../api';
 import type { Skill } from '../api';
 
 /**
@@ -23,7 +23,7 @@ export function useSkills(agentId: string | null, sessionId: string | null) {
 		setLoading(true);
 		setError(null);
 		try {
-			setSkills(await skillApi.list(agentId, sessionId));
+			setSkills(await workspaceApi.skill.list(agentId, sessionId));
 		} catch (e) {
 			setError(e as Error);
 		} finally {
@@ -39,7 +39,7 @@ export function useSkills(agentId: string | null, sessionId: string | null) {
 	const add = useCallback(
 		async (skillPath: string) => {
 			if (!agentId || !sessionId) throw new Error('No agent/session selected');
-			await skillApi.add(agentId, sessionId, { skill_path: skillPath });
+			await workspaceApi.skill.add(agentId, sessionId, { skill_path: skillPath });
 			await refetch();
 		},
 		[agentId, sessionId, refetch],
@@ -49,7 +49,7 @@ export function useSkills(agentId: string | null, sessionId: string | null) {
 	const remove = useCallback(
 		async (skillName: string) => {
 			if (!agentId || !sessionId) throw new Error('No agent/session selected');
-			await skillApi.remove(skillName, agentId, sessionId);
+			await workspaceApi.skill.remove(skillName, agentId, sessionId);
 			await refetch();
 		},
 		[agentId, sessionId, refetch],

--- a/examples/web_ui/frontend/tsconfig.app.json
+++ b/examples/web_ui/frontend/tsconfig.app.json
@@ -6,6 +6,7 @@
 		"module": "esnext",
 		"types": ["vite/client", "vite-plugin-svgr/client"],
 		"skipLibCheck": true,
+		"ignoreDeprecations": "6.0",
 
 		/* Bundler mode */
 		"moduleResolution": "bundler",

